lib/senkyoshi/canvas_course.rb

```diff
@@ -153,8 +162,8 @@ def upload_scorm_package(scorm_package, course_id, tmp_name)
     # Creates assignments from all previously uploaded scorm packages
     ##
     def create_scorm_assignments(scorm_packages, course_id, local)
-      scorm_packages.each do |pack|
-        create_scorm_assignment(pack, course_id, local)
+      scorm_packages.map do |pack|
```

Comment:
consider writing specs that don't depend on the return array? something like `expect create_scorm_assignment to be called 5 times`?